homeassistant/components/tuya/select.py

```diff
@@ -219,6 +219,30 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Robot Vacuum
+    # https://developer.tuya.com/en/docs/iot/fsd?id=K9gf487ck1tlo
+    "sd": (
+        SelectEntityDescription(
+            key=DPCode.CISTERN,
+            name="Water Tank Adjustment",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SelectEntityDescription(
+            key=DPCode.COLLECTION_MODE,
+            name="Dust Collection Mode",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SelectEntityDescription(
+            key=DPCode.SUCTION,
+            name="Suction",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SelectEntityDescription(
+            key=DPCode.MODE,
+            name="Mode",
+            entity_category=EntityCategory.CONFIG,
+        ),
+    ),
```

Comment:
Already part of the Vacuum entity

Reply:
Select Mode is not part of the vacuum entity.

The device has the ability to choose several modes not foreseen in the vacuum entity, this justifies the inclusion of the select Mode.


![](https://user-images.githubusercontent.com/31328123/144585275-f822f638-574e-4f9c-a95f-a159e65fb2cc.png)

Reply:
I think we can justify it, but we need to make sure we add translations for them (and not show raw values).

Reply:
Can you tell me the correct way to include the translations?

Reply:
More information can be found here: https://developers.home-assistant.io/docs/internationalization/core

Reply:
Done!
![image](https://user-images.githubusercontent.com/31328123/144594130-203fcb6f-33ad-4736-8218-de84c9911b84.png)
Thanks